frontend/src/modules/SaleProducts/SaleProducts.tsx

```diff
@@ -43,6 +44,7 @@ const SaleProducts: React.FC = () => {
               size={"small"}
               backgroundImage={item.backgroundImage}
               priceBeforeSale={item.priceBeforeSale}
+              isPreview={false}
```

Comment:
Consider making isPreview an optional field and setting its default value to false in the ProductCard component.